issue_metrics.py

```diff
@@ -175,9 +143,9 @@ def get_per_issue_metrics(
                 issue_with_metrics.mentor_activity = count_comments_per_user(
                     None,
                     issue,
-                    ignore_users,
                     None,
                     None,
+                    ignore_users,
```

Comment:
uh oh. This should have been caught by a test. Added one at 79a3995

Reply:
local linting caught it for me.  mypy ftw.  excellent idea to add tests.  thank you.

Reply:
Is mypy disabled in our super-linter? Wondering how this status check passed so we could merge.